scio-google-cloud-platform/src/it/scala/com/spotify/scio/bigquery/BigQueryIT.scala

```diff
@@ -44,7 +44,7 @@ class BigQueryIT extends AnyFlatSpec with Matchers {
   // Integration test with mock data
   // =======================================================================
 
-  "MockBigQuery" should "support mock data" in {
+  object MockBQData {
```

Comment:
As this is static, it can go in the `BigQueryIT` object

Reply:
do you mean instead of wrapping it in `MockBQData`, put it directly under `BigQueryIT`?

Reply:
you can keep the object grouping nested in  `BigQueryIT`. I think this is convenient.